cvmfs/server/cvmfs_server_mkfs.sh

```diff
@@ -204,6 +204,8 @@ cvmfs_server_mkfs() {
 
   # sanity checks
   check_repository_existence $name  && die "The repository $name already exists"
+  is_empty_repository_from_url $stratum0 ||
+      die "A manifest already exists at this url. Did you mean to use cvmfs_server import?"
```

Comment:
We may want to have a `force` flag or give instructions on how to really force making a new repo in the same space (manually remove the manifest & whitelist?)

Reply:
Yes, I think this usecase is too rare to warrant a force flag, but I can add a note regarding instructions (just deleting the manifest should be enough)